ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/query/QueryExecutorTest.java

```diff
@@ -187,6 +188,7 @@ public void setUp() {
         ImmutableSet.of(new QueryId("a")));
     when(pushQueryQuery.getRefinement())
         .thenReturn(Optional.of(RefinementInfo.of(OutputRefinement.CHANGES)));
+    when(pushQueryQuery.getSelect()).thenReturn(new Select(Collections.emptyList()));
```

Comment:
is this related to rowpartition/rowoffset?

Reply:
Indirectly, yes. Because this unit test uses a mocked KsqlConfig, previously the rowpartition/rowoffset feature flag defaulted to false. As a result, the push query validation [here](https://github.com/confluentinc/ksql/blob/f849bc92c6d0f60ec017947afa91da4a31d9967e/ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/util/ScalablePushUtil.java#L95) was never actually executed since the feature flag short-circuited the check early. Now that the feature flag is removed, the validation is executed, and it will NPE if the push query does not contain a `Select`.